apps/web/components/CardWithFeaturedItems/CardWithFeaturedItems.tsx

```diff
@@ -0,0 +1,114 @@
+import React from 'react'
+import { Box, Button, Hidden, Link, Tag, Text } from '@island.is/island-ui/core'
+import * as styles from './CardWithFeaturedItems.css'
+import { GetFrontpageQuery } from '@island.is/web/graphql/schema'
+import { LinkType, useLinkResolver } from '@island.is/web/hooks'
+
+type CardWithFeaturedItemsProps = {
+  heading: string
+  imgSrc: string
+  alt: string
+  href?: string | null
+  dataTestId?: string
+  seeMoreText: string
+  // eslint-disable-next-line @typescript-eslint/ban-ts-comment
+  // @ts-ignore make web strict
+  featuredItems: GetFrontpageQuery['getFrontpage']['lifeEvents'][0]['featured']
```

Comment:
```suggestion
  featuredItems: LifeEventPage['featured']
```

We should consider here relying also on the `LifeEventPage` type or, alternatively, use the `Featured` type directly from `'@island.is/web/graphql/schema'`